### PR TITLE
Bots: Add SQLite export

### PR DESCRIPTION
**Pull request by insunaa**
This PR provisionally adds bot support to the SQLite daily build.

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   schedule:
     # every day at 6am
     - cron:  '0 6 * * *'
+  push:
+    branches:
+      - "*"
```
Comment (insunaa) [inline .github/workflows/build-release.yml:9]: this was for testing

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -225,7 +239,7 @@ jobs:
       - name: Upload Snapshot
         uses: "marvinpinto/action-automatic-releases@latest"
         with:
-          repo_token: "${{ secrets.PUSH_RELEASE_TOKEN }}"
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (insunaa) [inline .github/workflows/build-release.yml:242]: not sure if this change was just needed to get it to work with my own repo

File: InstallFullDB.sh [hunk 3]
```diff
@@ -1470,10 +1470,10 @@ function apply_dev_content
...
 function apply_playerbots_db
 {
-  if [ "$PLAYERBOTS_DB" != "YES" ]; then
-    true
-    return
-  fi
+  # if [ "$PLAYERBOTS_DB" != "YES" ]; then
+  #   true
+  #   return
+  # fi
```
Comment (insunaa) [inline InstallFullDB.sh:1476]: this is obviously not ideal, but I also don't have any command line options with which I can force the bots to be built

File: .github/workflows/build-release.yml [hunk 4]
```diff
@@ -220,7 +231,7 @@ jobs:
 
       - name: Get Current Date
         id: date
-        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+        run: echo "date=$(date + '%Y-%m-%d')" >> $GITHUB_OUTPUT
```
Comment (insunaa) [inline .github/workflows/build-release.yml:234]: see https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by mostlikely4r** [comment]
This looks like a good addition. I'd like to see this merged.

**Pull request merged**